bcda/cli.go

```diff
@@ -62,6 +62,7 @@ const deleteThresholdHr = 8
 
 func importCCLF0(fileMetadata *cclfFileMetadata) (map[string]cclfFileValidator, error) {
 	if fileMetadata == nil {
+		fmt.Print("file CCLF0 not found")
```

Comment:
Like @knollfear mentioned in Slack, messages printed to stdout should be more like regular sentences (capitalized at the beginning at least)